src/main/resources/REST-README.md

```diff
@@ -0,0 +1,190 @@
+```
+  __  __ _                _____            __ _ _      
+ |  \/  (_)              |  __ \          / _(_) |     
+ | \  / |_  ___ _ __ ___ | |__) | __ ___ | |_ _| | ___ 
+ | |\/| | |/ __| '__/ _ \|  ___/ '__/ _ \|  _| | |/ _ \
+ | |  | | | (__| | | (_) | |   | | | (_) | | | | |  __/
+ |_|__|_|_|\___|_|  \___/|_|   |_|  \___/|_| |_|_|\___|
+  / ____| |           | |                              
+ | (___ | |_ __ _ _ __| |_ ___ _ __                    
+  \___ \| __/ _` | '__| __/ _ \ '__|                   
+  ____) | || (_| | |  | ||  __/ |                      
+ |_____/ \__\__,_|_|   \__\___|_|                      
+```
+
+# Project generator REST API
+
+This document summarizes examples of MicroProfile Starter web application REST API
+[microprofile/starter/1.0.0](https://app.swaggerhub.com/apis/microprofile/starter/1.0.0)
+
+Each MicroProfile version (mpVersion) offers a suite of specifications (specs) and is implemented
+by a set of application servers (supportedServers).
+
+In order to get a project zip generated one has to select at least the desired server (supportedServer).
+The latest mpVersion with all specs it offers is used by default then.
+
+# Get available MicroProfile versions
+
+```
+$ curl https://start.microprofile.io/api/1.0.0/mpVersion
```

Comment:
We should mention an alternative for Windows Users

Reply:
Chapter for Windows users added see [Windows users](https://github.com/Karm/microprofile-starter/blob/rest/src/main/resources/REST-README.md#windows-users)

Reply:
can we make the url to be shorter: https://start.microprofile.io/mpVersion
as I think we should always use the latest version of MP Starter?

Reply:
@Emily-Jiang I would like to leave the ```/api``` there so as it indicates what it is. 
It is true that we can move the API version from URL param to request header and offer the latest by default I guess... 
The most straightforward thing to do now would be to change "1.0.0" to "1" and leave it as URL param.

It would help me if I understand what is your motivation, is it programatic access, or overall curl usage simplicity? Or are you suggesting that versioning of REST API should be handled differently?

Thanks for clarification, Emily.

BTW, do you have any take on it @rdebusscher  ?